backend/app/adapter/db/url_integration_test.go

```diff
@@ -242,6 +243,123 @@ func TestURLSql_Create(t *testing.T) {
 	}
 }
 
+func TestURLSql_UpdateURL(t *testing.T) {
+	createdAt := mustParseTime(t, "2017-05-01T08:02:16-07:00")
+	now := time.Now()
+
+	testCases := []struct {
+		name           string
+		key            string
+		newAlias       string
+		newOriginalURL string
+		expireAt       *time.Time
+		tableRows      []urlTableRow
+		hasErr         bool
+		expectedURL    entity.URL
+	}{
+		{
+			name: "alias not found",
+			key:  "does_not_exist",
+			tableRows: []urlTableRow{
+				urlTableRow{
+					alias:     "220uFicCJj",
+					longLink:  "https://www.google.com",
+					createdAt: &createdAt,
+				},
+			},
+			hasErr:      true,
+			expectedURL: entity.URL{},
+		},
+		{
+			name: "alias already exists",
+			key:  "220uFicCja",
+			tableRows: []urlTableRow{
+				urlTableRow{
+					alias:     "220uFicCJj",
+					longLink:  "https://www.google.com",
+					createdAt: &createdAt,
+				},
+				urlTableRow{
+					alias:     "efpIZ4OS",
+					longLink:  "https://gmail.com",
+					createdAt: &createdAt,
+				},
+			},
+			hasErr:      true,
+			expectedURL: entity.URL{},
+		},
+		{
+			name:     "valid new alias",
+			key:      "220uFicCJj",
+			newAlias: "GxtKXM9V",
+			tableRows: []urlTableRow{
+				urlTableRow{
+					alias:     "220uFicCJj",
+					longLink:  "https://www.google.com",
+					createdAt: &createdAt,
+				},
+			},
+			hasErr: false,
+			expectedURL: entity.URL{
+				Alias:       "GxtKXM9V",
+				OriginalURL: "https://www.google.com",
+				UpdatedAt:   &now,
+			},
+		},
+		{
+			name:           "valid new long link",
+			key:            "220uFicCJj",
+			newOriginalURL: "https://www.facebook.com",
+			tableRows: []urlTableRow{
+				urlTableRow{
+					alias:     "220uFicCJj",
+					longLink:  "https://www.google.com",
+					createdAt: &createdAt,
+				},
+			},
+			hasErr: false,
+			expectedURL: entity.URL{
+				Alias:       "220uFicCJj",
+				OriginalURL: "https://www.facebook.com",
+				UpdatedAt:   &now,
+			},
+		},
```

Comment:
I think this one is unnecessary unless we provide an test case for invalid long link.